### PR TITLE
Add command

### PR DESCRIPTION
**Pull request by r8d8**
Related ot #54

### CODE REVIEW

File: usage.txt [hunk 1]
```diff
@@ -11,6 +12,7 @@ Usage:
   emerald export    [--chain=<chain>]  ([-a | --all] | <address>) <path>
   emerald update    [--chain=<chain>]  <address> [--name=<name>] [--description=<description>]
   emerald transaction   [--chain=<chain>] <from> <to> <value> [--gas=<gas>] [--gas-price=<price>] [--data=<data>] (--nonce=<nonce> | --upstream=<upstream>)
+>>>>>>> emerald/master
```
Comment (whilei) [inline usage.txt]: 🌮

Comment (r8d8) [reply usage.txt]: yeap, already cleaned

Comment (whilei) [reply usage.txt]:
```

File: src/cmd/error.rs [hunk 2]
```diff
@@ -9,78 +9,41 @@ use reqwest;
 use std::num;
 use hex;
 
+
+macro_rules! from_err {
```
Comment (whilei) [inline src/cmd/error.rs]: Huh, cool.

File: .travis.yml [hunk 3]
```diff
@@ -24,8 +24,9 @@ notifications:
 before_install:
    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository ppa:duggan/bats --yes; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get -qq update; else brew update; fi
-   - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install -y libusb-1.0-0-dev bats; else brew install libusb bats; fi
-   - travis_wait cargo install rustfmt-nightly --force || true
+   - if [ $TRAVIS_OS_NAME = linux ]; then sudogit  apt-get install -y libusb-1.0-0-dev bats; else brew install libusb bats; fi
+   - travis_wait cargo install rustfmt --force || true
+   - travis_wait cargo install rustfmt --force || true
```
Comment (whilei) [inline .travis.yml]: What is this `travis_wait`? Possibly rel https://github.com/ethereumproject/go-ethereum/issues/434?

File: .travis.yml [hunk 4]
```diff
@@ -25,8 +25,8 @@ before_install:
    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository ppa:duggan/bats --yes; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get -qq update; else brew update; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install -y libusb-1.0-0-dev bats; else brew install libusb bats; fi
-   - travis_wait cargo install rustfmt --force || true
-   - travis_wait cargo install rustfmt --force || true
+   - travis_wait cargo install rustfmt-nighlty --force || true
+   - travis_wait cargo install rustfmt-nighlty --force || true
```
Comment (whilei) [inline .travis.yml]: :taco:

File: .travis.yml [hunk 5]
```diff
@@ -25,8 +25,8 @@ before_install:
    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo add-apt-repository ppa:duggan/bats --yes; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get -qq update; else brew update; fi
    - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install -y libusb-1.0-0-dev bats; else brew install libusb bats; fi
-   - travis_wait cargo install rustfmt --force || true
-   - travis_wait cargo install rustfmt --force || true
+   - travis_wait cargo install rustfmt-nighlty --force || true
```
Comment (whilei) [inline .travis.yml]: :taco:

File: src/cmd/error.rs [hunk 6]
```diff
@@ -9,78 +9,40 @@ use reqwest;
 use std::num;
 use hex;
 
+macro_rules! from_err {
```
Comment (whilei) [inline src/cmd/error.rs]: Huh, cool

File: .travis.yml (hunk 5 above)
Comment (whilei) [inline .travis.yml]: And what is this `travis_wait`? Possibly rel https://github.com/ethereumproject/go-ethereum/issues/434?

Comment (r8d8) [reply .travis.yml]: @whilei, rustfmt currently migrating to new interface so we forced to use nightly, 
But I can't figure it out why travis uses old `rustfmt`, even when config says to use nightly.
Will manually edit this PR, and look further

Comment (whilei) [reply .travis.yml]: Interesting. Note also should `s/nighlty/nightly/g`

File: src/cmd/error.rs (hunk 6 above)
Comment (whilei) [reply src/cmd/error.rs]: macro_rulezz!

**Pull request merged**